addon/mixins/spread.js

```diff
@@ -108,9 +108,11 @@ export default Mixin.create({
    * @param {object} spreadableHash - the hash to spread
    */
   _defineSpreadProperties (spreadProperty, spreadableHash) {
+    const supportedSpreadableTypes = A(['instance', 'object'])
+
     assert(
       `${spreadProperty} requires an Ember object or primitive object`,
-      ['instance', 'object'].includes(typeOf(spreadableHash))
+      supportedSpreadableTypes.includes(typeOf(spreadableHash))
```

Comment:
That's still not using a feature of the the language that's fully supported by browsers, for the sole reason that it's not supported by phantom though.